### PR TITLE
Feature/red 9 grade override

### PR DESCRIPTION
**Pull request by red-tt-bear**

### CODE REVIEW

File: src/utilities/renderer-helper.ts [hunk 1]
```diff
@@ -179,7 +179,7 @@ class RendererHelper {
     async getProblem({
         sourceFilePath,
         problemSource,
-        problemSeed,
+        problemSeed = 666,
```
Comment (PanopticaRising) [inline src/utilities/renderer-helper.ts:182]: Why is this defaulted to 666 here and defaulted to [here](https://github.com/rederly/backend/pull/134/files#diff-1fd04e20c23b5de82549b9541a8df404R878)?

File: src/features/courses/course-repository.ts [hunk 2]
```diff
@@ -445,6 +446,25 @@ class CourseRepository {
...
+        try {
+            const updates = await StudentGrade.update(options.updates, {
+                where: {
+                    ...options.where,
+                    active: true
+                },
+                returning: true,
+            });
+            return {
+                updatedCount: updates[0],
```
Comment (PanopticaRising) [inline src/features/courses/course-repository.ts:460]: Do we need to document the mapping of these array returns?

**Review by PanopticaRising — APPROVED** [review]

File: src/utilities/renderer-helper.ts (hunk 1 above)
Comment (doombot-exe) [reply src/utilities/renderer-helper.ts:182]: renderer will always default a missing problemSeed to 666 regardless

Comment (red-tt-bear) [reply src/utilities/renderer-helper.ts:182]: I could remove it, probably makes sense, the doc had it labeled as required so I included it 😆

File: src/features/courses/course-repository.ts (hunk 2 above)
Comment (red-tt-bear) [reply src/features/courses/course-repository.ts:460]: Not sure what you're asking, but:
1. The response is typed
2. This is not actually an array, it's a tuple

File: src/utilities/renderer-helper.ts (hunk 1 above)
Comment (PanopticaRising) [reply src/utilities/renderer-helper.ts:182]: I was just confused cause you changed the 666 above to null, so I was wondering why the inconsistency.

**Pull request merged**